### PR TITLE
Add proposed COVID-19 measures to calendar page

### PR DESCRIPTION
**Pull request by valentine**

### CODE REVIEW

File: contents/calendar/_index/main.markdown [hunk 1]
```diff
@@ -25,6 +25,21 @@ events (excluding HackerspaceSG’s cleaning every Tuesday, of course).
 
 # Hosting Events
 
+## COVID-19 measures
+
+In view of the current outbreak of the Coronavirus Disease 2019 (COVID-19), HackerspaceSG strongly encourages event organisers to take the following measures:
+
+* Ask visitors to fill out a travel and health declaration form, and to include their personal information for contact tracing purposes, before the event.
```
Comment (robertsky) [inline contents/calendar/_index/main.markdown]: do we need to add a warning that false declaration may have legal consequences here in Singapore? the crowd that some events attract may have a higher aversion to giving out private information, be it due to differences in laws, culture, etc. see: https://www.tnp.sg/news/singapore/travellers-false-declarations-can-be-prosecuted-moh and https://www.channelnewsasia.com/news/singapore/covid19-coronavirus-charged-false-info-moh-contact-tracing-12471894

File: contents/calendar/_index/main.markdown [hunk 2]
```diff
@@ -25,6 +25,21 @@ events (excluding HackerspaceSG’s cleaning every Tuesday, of course).
...
+* Attendees who have been to mainland China, Iran, northern Italy, the Republic of Korea, or [any other such regions highlighted by the Singapore Ministry of Health](https://www.gov.sg/article/covid-19-travel-restrictions-for-foreign-visitors-entering-singapore), within the last 14 days, should be asked to not participate in the event.
+* Attendees who exhibit mild respiratory infection symptoms should wear a mask at all times during the event.
+* Attendees who exhibit serious respiratory infection symptoms should be asked to leave the HackerspaceSG premises.
+* Full discretion falls on either the event organiser, or any other HackerspaceSG member in the premises at the time, to enforce this.
+* Handshaking and other forms of physical contact should be discouraged.
+
+We encourage everyone to be vigilant, socially responsible, and observe good personal hygiene. Please seek [medical assistance](https://www.flugowhere.gov.sg/?searchValue=208576&tab=0) promptly if you are unwell.
+
+Thank you for your cooperation and understanding.
+
```
Comment (robertsky) [inline contents/calendar/_index/main.markdown]: In view of the fluid changes of COVID-19, we should add a last updated date for this section. Let's be frank about this, an average website visitor will not do a git blame to see when the section was last updated.

Comment (valentine) [reply contents/calendar/_index/main.markdown]: Added: https://github.com/hackerspacesg/hackerspace.sg/commit/06db371d9b5a6754bf2bb996e98d45c37a9f0a7d#diff-c3781d18a09c98f50f516cb3129cd250R30

File: contents/calendar/_index/main.markdown (hunk 1 above)
Comment (valentine) [reply contents/calendar/_index/main.markdown]: Added on [the form](https://docs.google.com/forms/u/2/d/e/1FAIpQLScGBotz1GKpaTqvZEAw7HecUSSGfBhbEEVxKeTP0kuPSURSAA/viewform).

Comment (robertsky) [reply contents/calendar/_index/main.markdown]: > All collected information will be destroyed when the authorities lower the DORSCON level to Yellow.

Suggest changing this to 
> All collected information will be destroyed after 90 days or when the authorities deem that contacting tracing is no longer needed, whichever is earlier.

Lowering to Yellow may not reduce the authorities' want for contact tracing. After all, DORSCON could be lowered to Yellow even when the spread is still ongoing. https://www.channelnewsasia.com/news/singapore/dorscon-level-covid19-transmits-worldwide-coronavirus-12469484. As for 90 days limit, it is a time limit that I think is just a bit longer than what authorities would ask for given that the incubation is up to 14 days (some say 20-30+ days), and with allowance for backfilling, maybe 2 months of prior data?

Comment (robertsky) [reply contents/calendar/_index/main.markdown]: also name field to be compulsory

Comment (valentine) [reply contents/calendar/_index/main.markdown]: Thanks, changed.

**Pull request merged**